src/main/asciidoc/http.adoc

```diff
@@ -1096,7 +1096,7 @@ Here's an example:
 The redirection policy is as follow
```

Comment:
this section should be removed because it's incorrect and better explained in the documentation above (see the comment)